README.md

```diff
@@ -1,9 +1,125 @@
-Freesound is a project by the Music Technology Group (MTG, http://www.mtg.upf.edu), Universitat Pompeu Fabra
+![Freesound](media/images/logo_bw.png)
 
-All the source code in this repository is licensed as GNU Affero (see _LICENSE folder).
+This repository contains the source code of the [Freesound](https://freesound.org) website.
 
-For instructions on how to set up Freesound for development, [see the wiki](https://github.com/MTG/freesound/wiki/Development-setup)
-
-For notes about development, see DEVELOPERS.md
+Freesound is a project by the [Music Technology Group](http://www.mtg.upf.edu) (MTG), [Universitat Pompeu Fabra](http://upf.edu) (UPF).
 
 [![Build Status](https://travis-ci.org/MTG/freesound.svg?branch=master)](https://travis-ci.org/MTG/freesound)
+
+
+## License
+
+All the source code in this repository is licensed as GNU Affero. Some of the dependencies might have their own licenses. See the [_LICENSE](https://github.com/MTG/freesound/tree/master/_LICENSE) folder for more details.
+
+
+## Authors
+
+For a list of authors please check out the [contributors](https://github.com/MTG/freesound/graphs/contributors) page.
+
+
+## Development
+
+Freesound is composed by a numnber of different services which can be run and orchestrated using Docker . The main service is provided by the `web` container which runs the Freesound Django application. Check out [this blog post](https://opensource.creativecommons.org/blog/entries/freesound-intro/) for some information about the Freesound technology stack. If you're to do development for Freesound, please checkout the [DEVELOPERS](https://github.com/MTG/freesound/blob/master/DEVELOPERS.md) file for some guidelines. 
```

Comment:
extra space before dot.
Should we consider wrapping lines in readme?

Reply:
If you're going to do development on Freesound, please check the

Reply:
I fixed the space and phrasing. I think no need to wrap.